README.md

```diff
@@ -101,6 +101,16 @@ Examples:
     curf.format(1234.56, 'USD') #=> "$1,234.56"
 ```
 
+```ruby
```

Comment:
Please insert a "Time formatting" header above this so that the sections in this file is consistent. Right now this is under the Number/currency formatting header.